docs/markdown/authoritative/upgrading.md

```diff
@@ -6,6 +6,8 @@ Please upgrade to the PowerDNS Authoritative Server 4.0.0 from 3.4.2+. See the [
 
 ## Changed options
 
+ * `experimental-lua-policy-script` and feature has been dropped completely. We invite you to to use [(PowerDNS dnsdist)[http://dnsdist.org] instead.
```

Comment:
s/and feature/option and feature/

s/to to use/to use/ and also i think the first [ needs to go